NEWS.md

```diff
@@ -1,40 +1,43 @@
 # ggplot2 (development version)
 
+* `stat_smooth()` user `REML` by default, if `method = "gam"` and
+  `gam`'s method is not specified (@ikosmidis, #2630).
+
 * `expand_scale()` was deprecated in favour of `expansion()` for setting
```

Comment:
I think something has gone wrong with this edit as its made a bunch of whitespace changes to the file. 

To help me figure out the source of the problem, are you using RStudio?

Reply:
I am using ESS. Any advice on how I can prevent this next time?